packages/components/radio/src/radio.css

```diff
@@ -0,0 +1,81 @@
+/* General........................................................ */
+
+.radio-container {
+	display: block;
+	position: relative;
+	padding-left: var(--ts-unit-plus);
+	margin: var(--ts-unit-half);
+	cursor: pointer;
+	font-size: var(--ts-fontsize);
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+	&:hover {
+		& span {
+			background-color: var(--ts-color-gray-lighter);
+		}
+	}
+	& input {
+		position: absolute;
+		opacity: 0;
+		cursor: pointer;
+		height: 0;
+		width: 0;
+	}
+	& span {
+		position: absolute;
+		top: 0;
+		left: 0;
+		height: var(--ts-unit);
+		width: var(--ts-unit);
+		border: var(--ts-border);
+		border-radius: 50%;
+		&:after {
+			content: '';
+			position: absolute;
+			display: none;
+			top: var(--ts-unit-quarter);
+			left: var(--ts-unit-quarter);
+			width: var(--ts-unit-half);
```

Comment:
Based on the design the inner part should be `8px` not `10px`.
@mihneaz can you check if it's the right design?
https://share.goabstract.com/a5fb0def-1ffc-4f0b-9bc7-42d596ae4ed3?mode=build&sha=d180b45775e2891dbe98135a98003363cdb240df